iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -31,6 +31,45 @@
 //Defined private method inside TiBindingRunLoop.m (Perhaps to move to .c?)
 void TiBindingRunLoopAnnounceStart(TiBindingRunLoop runLoop);
 
+typedef NS_ENUM(NSInteger, FileStatus) {
+  DoesntExist,
```

Comment:
Only code style, but usually in ObjC, the enumeration cases are prefixed with the enumeration itself, e.g. FileStatusDoesntExist.